app/src/importimageseqdialog.cpp

```diff
@@ -58,9 +58,15 @@ void ImportImageSeqDialog::setupLayout()
 
     hideInstructionsLabel(true);
 
-    if (mFileType == FileType::GIF) {
+    switch (mFileType)
+    {
+    case FileType::GIF:
         setWindowTitle(tr("Import Animated GIF"));
-    } else {
+        break;
+    case FileType::IMAGE_SEQUENCE:
+        setWindowTitle(tr("Import animated image"));
+        break;
+    default:
         setWindowTitle(tr("Import image sequence"));
```

Comment:
I think you accidentally switched these around, now the title will be "import animated image" for an image sequence, and "import image sequence" for animated images 🙃